### PR TITLE
promo code applied for login 50%off on bid amount

### PR DESCRIPTION
**Pull request by siddharth05092004**

### CODE REVIEW

**Review by souravjaisingh — APPROVED** [review]

File: VibeSyncApp/ClientApp/src/components/PaymentIndex.js [hunk 1]
```diff
@@ -220,7 +221,19 @@ function PaymentIndex() {
     }, [isSpecialAnnouncement, rowData.minimumBid, rowData.minimumBidForSpecialRequest, amount]);
 
 
-
+    useEffect(()=>{
+      if(localStorage.getItem('userId')!==null && localStorage.getItem('userId')!==0){
+        try{
+          let rowData_to_change = rowData;
+          rowData_to_change.minimumBid = Math.round(rowData.minimumBid/2)
```
Comment (souravjaisingh) [inline VibeSyncApp/ClientApp/src/components/PaymentIndex.js:228]: We don't have to make minimum bid / 2. It should stay same. A new label should appear however, just like GST, which shows the discount applied. And Amount on Pay button should reflect that.

**Review by souravjaisingh — CHANGES_REQUESTED** [review]
We don't have to make minimum bid / 2. It should stay same. A new label should appear however, just like GST, which shows the discount applied. And Amount on Pay button should reflect that.

**Review by bhavikmittal — CHANGES_REQUESTED** [review]
Address the comments

**Comment by siddharth05092004** [comment]
Will do it today.

**Pull request closed without merging**